meetings/2021-11-22.md

```diff
@@ -0,0 +1,93 @@
+* eP: It would be nice to have some way of linking to the rendered preview of that PR for discussion
+* AC: We could do this manually if automating is too much of a hassle.
+* eP: We can move on with the discussion until we are happy enough with it and open it for broader discussion, and only then render it.
+* AC: It would be good to have a clear horizon for the public feedback. We should be conscious about the fact that the Christmas holidays are coming up and that a lot of people are going to be out at this point.
+* eP: Worst case scenario, if feedback is too negative with compelling arguments, we can always revert the change. We could agree on merging this this week, and open a 2-week window.
+* AC: This sounds like a good plan. This is also a good first step to define a step forward, because we stop specifying the role of the OIDC issuer issuing Access Token, but we don't prevent it altogether, which maintians backwards compat
+* eP: The role of the AS is very clear in the current spec. Do we want to add back some text about where the access token is coming from ?
+* AC: No, we should only describe what we want for the future, but not preventing the current approach helps ramping up the ecosystem to this flow.
+* eP: Most likely the clients will be able to detect if the new flow is supported or not.
+* eP: For the test suite, will the code only test the OpenID provider, or should the test suite cover each individual conformance class ?
+* AC: The latter would be the direction I'd like to go in. Currently it only focuses on the OpenID provider, but eventually being able to test conformance of Openid provider, AS, Client identifier, and RS. Not sure how to test clients.
+* eP: Maybe exposing a test server endpoint. Being able to test conformance of client ID documents would be a nice first step, these are minimal requirements.
+* AC: That sounds great. The test currently tests its own client ID document, but it would be good to go towards testing a permutation of parameters.
+* eP: Being able to split the conformances tests by conformance class would also make it easier to implement because tests only make sense in the context of a single class.
+
+
+### Specify the client authentication method when using a Client Identifier
+
+* NS: Using ClientID document prevents us from needing client secrets. Does the `client_id` need to be added to request body? Should it be mentioned in the spec?
+* AC: Good question. We discussed it but didn't take action on it. The main problem with `client_id` going in Authorization: header is the presence of the column after the protocol, because the Basic auth scheme splits on column. You just need to URL encode `client_id` if you are using BASIC auth. It may make sense to add non-normative text about that case of URL encoding `client_id` (URL).
```

Comment:
```suggestion
* AC: Good question. We discussed it but didn't take action on it. The main problem with `client_id` going in `Authorization:` header is the presence of the column after the protocol, because the Basic auth scheme splits on column. You just need to URL encode the `client_id` if you are using BASIC auth. It may make sense to add non-normative text about that case of URL encoding the `client_id` (URL).
```